saleor/graphql/attribute/mutations/attribute_value_delete.py

```diff
@@ -23,10 +23,22 @@ class Arguments:
         )
 
     class Meta:
+        auto_permission_message = False
         model = models.AttributeValue
         object_type = AttributeValue
-        description = "Deletes a value of an attribute."
-        permissions = (ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,)
+        description = (
+            "Deletes a value of an attribute.\n\n"
+            "Depending on the attribute type, "
+            "it requires different permissions to delete:\n"
+            "-`PRODUCT_TYPE`: Requires one of the following permissions: "
+            "`MANAGE_PRODUCTS` or `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`.\n"
+            "-`PAGE_TYPE`: `MANAGE_PRODUCTS` or `MANAGE_PAGES` or "
+            "`MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`.\n"
+            "\n\nDEPRECATED in Saleor 4.0, for attribute type:\n"
```

Comment:
It is deprecated in the current Saleor version, not 4.0 right? In 4.0 this will be removed.